crates/stdsimd-test/src/lib.rs

```diff
@@ -136,17 +136,16 @@ pub fn assert(fnptr: usize, fnname: &str, expected: &str) {
         .ok()
         .map_or_else(
             || match expected {
-                // cpuid returns a pretty big aggregate structure so exempt it
-                // from the slightly more restrictive 22
-                // instructions below
+                // `cpuid` returns a pretty big aggregate structure, so exempt it
+                // from the slightly more restrictive 22 instructions below.
                 "cpuid" => 30,
 
-                // Apparently on Windows LLVM generates a bunch of
+                // Apparently, on Windows, LLVM generates a bunch of
                 // saves/restores of xmm registers around these
-                // intstructions which blows the 20 limit
-                // below. As it seems dictates by Windows's abi
-                // (I guess?) we probably can't do much
-                // about it...
+                // intstructions which exceeds the 20 limit
```

Comment:
shouldn't this be "exceed" ? we are talking about "the instructions" and not "LLVM"

Reply:
I think the implicit noun here is "bunch"?